elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -7,10 +7,9 @@ $pfe-cta--BackgroundColor--focus: rgba(40, 151, 240, 0.2);
 $pfe-cta--Color--fallback: #003366;
 
 // Arrow animation
-$left: 0.2rem;
-$right: 0.2rem;
-$arrow-basic: 0 $right 0 $left;
-$arrow-hover: 0 0 0 #{$right + $left};
+$h-spacing: 3px;
```

Comment:
@starryeyez024 What does h stand for?

Reply:
horizontal, i can spell it out if you prefer